pupil_src/shared_modules/calibration_routines/single_marker_calibration.py

```diff
@@ -9,6 +9,11 @@
 ---------------------------------------------------------------------------~(*)
 '''
 
+'''
+Reference:
+Santini, Thiago, Wolfgang Fuhl, and Enkelejda Kasneci. "CalibMe: Fast and Unsupervised Eye Tracker Calibration for Gaze-Based Pervasive Human-Computer Interaction." Proceedings of the 2017 CHI Conference on Human Factors in Computing Systems. ACM, 2017.
```

Comment:
@papr should this be be broken into a couple of lines, so that it does not create one long line of text here?

Reply:
This will be moved to the docs.

Reply:
Yes, that is a good idea.